### DISCUSSION TITLE
Inventory Handling

### DISCUSSION THREAD

**Discussion by SC-HARSH** [ORelio/Minecraft-Console-Client#2043] [Q&A]
Heyy,
I am new to MCC,
I need the command for this
I want to left-click on the compass
![image](https://user-images.githubusercontent.com/69623681/172653740-2b77e9e7-0dff-4d07-8c49-d16c41abe729.png)
Then I want to click on the pickaxe
![image](https://user-images.githubusercontent.com/69623681/172653434-ef8bd5f6-735a-4dc6-9a73-2c2163a760bb.png)
Please Help asap

**Comment by ORelio** [comment]
Use /inventory list/click. See #1167

**Reply by SC-HARSH** [reply]
It would be great if you send the command as there are screenshots of that the window
Thank you

**Reply by ORelio** [reply]
`/inventory container list`
`/inventory container click <slot>`
